test/fixtures/comprehensions/assignment-expr/actual.js

```diff
@@ -0,0 +1,3 @@
+[for idx i in Array(10):
+  now x = f(i)
```

Comment:
Actually, with `now`, it's perfectly fine to leave it as `arr.push(x = f(i));` – the case I was worried about was the auto-const `x = f(i)`